README.md

```diff
@@ -7,8 +7,8 @@ OpenEmbedded layer for latest [Node.js](https://nodejs.org/ "Node.js") releases.
 
 ## Node.js releases
 
- * ![Current 7](https://img.shields.io/badge/Node.js%20Current-7.10.0-green.svg)
- * ![LTS 6](https://img.shields.io/badge/Node.js%20LTS-6.10.3-blue.svg)
+ * ![Current 8](https://img.shields.io/badge/Node.js%20Current-8.4.0-green.svg)
```

Comment:
Should this actually be LTS: 8.9.4? Current is 9.x.x (even though we still don't support it here)